Automattic/wp-calypso
# Update the G Suite checkout nudge to show Google Workspace instead

#### Changes proposed in this Pull Request

* Since G Suite is rebranded to Google Workspace we need to prepare for that switchover. Here's an update to the checkout G Suite nudge to show Google Workspace instead

It looks like this:

<img width="763" alt="Screenshot 2021-01-22 at 21 52 40" src="https://user-images.githubusercontent.com/1355045/105539525-899a3c80-5cfd-11eb-8938-02a335ecfe8c.png">

#### Testing instructions

* Open `checkout/{blogdomain}/with-gsuite/{domain}/{receiptID}` and verify that Google Workspace is displayed
